contracts/Blast_SpokePool.sol

```diff
@@ -32,6 +32,17 @@ interface IBlast {
     function claimMaxGas(address contractAddress, address recipientOfGas) external returns (uint256);
 }
 
+interface IUSDBL2Bridge {
```

Comment:
For review, the Blast docs are here: https://docs.blast.io/building/contracts.
The relevant contract is the L2BlastBridge, here: https://blastscan.io/address/0x4300000000000000000000000000000000000005#writeProxyContract#F2